docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -2,7 +2,10 @@
 
 Trino includes a native implementation to access [Amazon
 S3](https://aws.amazon.com/s3/) and compatible storage systems with a catalog
-using the Delta Lake, Hive, Hudi, or Iceberg connectors.
+using the Delta Lake, Hive, Hudi, or Iceberg connectors. While the exchange
+manager is designed to support S3-compatible storage systems, not all such
```

Comment:
@electrum @findinpath @ebyhr .. do we want to clarify that we test with S3 and I think also MinIO?

Reply:
Sounds good .. can you address that @michaeleby1